### ISSUE TITLE
composer install error packages

### ISSUE THREAD

**Issue by thienhv** [LavaLite/cms#21]
Hi, i have problem when install lavalite. Pls, helpme 
Problem 1
    - Conclusion: don't install illuminate/support 5.0.x-dev
    - Conclusion: don't install illuminate/support 4.2.x-dev
    - Conclusion: don't install illuminate/support v4.2.17
    - Conclusion: don't install illuminate/support v4.2.16
    - Conclusion: don't install illuminate/support v4.2.12
    - Conclusion: don't install illuminate/support v4.2.9
    - Conclusion: don't install laravel/framework 4.2.x-dev
    - Conclusion: don't install laravel/framework v4.2.17
    - Conclusion: don't install illuminate/support v5.0.26
    - Conclusion: don't install illuminate/support v4.2.8
    - Conclusion: don't install illuminate/support v4.2.7
    - Conclusion: don't install illuminate/support v4.2.6
    - Conclusion: don't install illuminate/support v4.2.5
    - Conclusion: don't install illuminate/support v4.2.4
    - Conclusion: don't install laravel/framework v4.2.16
    - Conclusion: don't install laravel/framework v4.2.15
    - Conclusion: don't install illuminate/support v5.0.25
    - Conclusion: don't install illuminate/support v4.2.3
    - Conclusion: don't install illuminate/support v4.2.2
    - Conclusion: don't install illuminate/support v4.2.1
    - Conclusion: don't install illuminate/support v4.2.0-BETA1
    - Conclusion: don't install laravel/framework v4.2.14
    - Conclusion: don't install laravel/framework v4.2.13
    - Conclusion: don't install laravel/framework v4.2.12
    - Conclusion: don't install illuminate/support v5.0.22
    - Conclusion: don't install laravel/framework v4.2.11
    - Conclusion: don't install laravel/framework v4.2.10
    - Conclusion: don't install laravel/framework v4.2.9
    - Conclusion: don't install laravel/framework v4.2.8
    - Conclusion: don't install laravel/framework v4.2.7
    - Conclusion: don't install laravel/framework v4.2.6
    - Installation request for lavalite/user 2.2.* -> satisfiable by lavalite/user[2.2.0].
    - Installation request for greggilbert/recaptcha dev-master -> satisfiable by greggilbert/recaptcha[dev-master].
    - Installation request for laravel/framework 4.2.* -> satisfiable by laravel/framework[4.2.x-dev, v4.2.0, v4.2.0-BETA1, v4.2.1, v4.2.10, v4.2.11, v4.2.12, v4.2.13, v4.2.14, v4.2.15, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - lavalite/user 2.2.0 requires illuminate/support 4.2.* -> satisfiable by laravel/framework[4.2.x-dev, v4.2.0, v4.2.0-BETA1, v4.2.1, v4.2.10, v4.2.11, v4.2.12, v4.2.13, v4.2.14, v4.2.15, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9], illuminate/support[4.2.x-dev, v4.2.0-BETA1, v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.0
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.0-BETA1
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.1
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.2
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.3
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.4
    - don't install illuminate/support v5.0.0|don't install laravel/framework v4.2.5
    - greggilbert/recaptcha dev-master requires illuminate/support 5.0.* -> satisfiable by illuminate/support[5.0.x-dev, v5.0.0, v5.0.22, v5.0.25, v5.0.26, v5.0.4].
    - Conclusion: don't install illuminate/support v5.0.4

Ps: My composer version used:
Composer version 1.0-dev (4aa431025b38d0783d47984a6f3a0c02b186fa58) 2015-04-15 01:51:00

**Comment by eckenroed** [comment]
I too ran into this issue. I am using a Mac. I had to do 2 things in order to get it ton install correctly...

1) I had to run the command as "sudo" to give it the proper access to create the directories
2) I had to request a specific version (2.2.1) instead of (2.2.*). 

So running 

```
sudo composer create-project LavaLite/cms path 2.2.1
```

seemed to do the trick. We'll, I ran into another issue with Mcrypt not being installed, but then I realized I should have been doing this through my Vagrant box and I wouldn't have had any of these issues. 

That I am aware of there is literally no way to get Mcrypt to work correctly on the Mac, (whether using native, Lamp, Mamp, xamp, etc..). You must run a VM (Vagrant via VirtualBox..both are free).
